docs/manual/guides/local-installation/ddev.en.md

```diff
@@ -26,119 +26,87 @@ more information about installing these programs.
-
-sudo apt update && sudo apt install -y ddev
+ddev composer create contao/managed-edition:5.3
 ```
 
-If necessary, install the local SSL certificates after the installation (afterwards restart your browser).
+After installation, the database access data must be entered in the `.env.local`. At the same time, we also set up Mailpit directly.
 
-```shell
-mkcert -install
+```env
+APP_ENV=prod
+DATABASE_URL=mysql://db:db@db:3306/db
+MAILER_DSN=smtp://localhost:1025
 ```
 
-Installation Update
+Then create the database:
 
 ```shell
-sudo apt update && sudo apt upgrade
+ddev exec "bin/console contao:migrate"
 ```
 
-
-## Create project
-
-Open the console of your choice, create the desired directory and then change to it.
+Create backend user:
 
 ```shell
-mkdir -p ~/projects/contao/contao-ddev && cd ~/projects/contao/contao-ddev
+ddev exec "bin/console contao:user:create"
 ```
 
-Create the DDEV configuration with:
+Call up the project in the browser:
 
 ```shell
-ddev config
+ddev launch
 ```
 
-Make DDEV settings, select `php` as __Project Type__ in any case. Leave the __Docroot Location__ empty for now, because there is no `public` folder for new installations and DDEV can not start then.
-
-```shell
-ddev start
-```
+{{% notice note %}}
 
-For installation via console it is easiest to go via ssh into the container.
+With `ddev launch contao` you get directly to the administration.
 
-```shell
-ddev ssh
-```
+{{% /notice %}}
 
-```shell
-composer create-project contao/managed-edition contao 4.13
-```
+The `ddev` binary is not available in the container, so first switch to the host console with `exit`.
 
-In the `.ddev/config.yml` now adjust the docroot and restart ddev.
+## Additional information
 
-```yml
-docroot: "contao/public"
-```
+`ddev start` starts the project, `ddev stop` ends it. Make sure beforehand that you have changed to the project folder.
 
-To use Apache instead of NGINX, change your entry `webserver_type: nginx-fpm` to `apache-fpm`.
+`ddev poweroff` can stop all started projects/containers from any directory.
 
-```yml
-webserver_type: apache-fpm
-```
+With `ddev ssh` you can switch to the shell of the container and work on the console. The `ddev` binary is not available in the container, so first switch to the host console with `exit`.
 
-The `ddev` binary is not available in the container, so first switch to the host console with `exit`.
+`ddev describe` gives an overview of the services available in the project and how to access them.
 
-```shell
-ddev restart
-```
+`ddev xdebug on` starts XDebug. [Information about the IDE setup](https://ddev.readthedocs.io/en/latest/users/debugging-profiling/step-debugging/#ide-setup)
 
-A database already exists in DDEV. The connection data for the installation are:
+{{% notice note %}}
+If you are a Windows user using the "Git Bash" as a console, it may be necessary, depending on your "Git for Windows" configuration, to prepend the command winpty (e.g.: winpty ddev ssh).
```

Comment:
```suggestion
If you are a Windows user using the "Git Bash" as a console, it may be necessary, depending on your "Git for Windows" configuration, to prepend the command `winpty` (e.g.: `winpty ddev ssh`).
```